rollup.config.js

```diff
@@ -8,10 +8,14 @@ import replace from 'rollup-plugin-replace';
 import strip from 'rollup-plugin-strip';
 import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
 
+const pkg = require('./package.json');
+
 const input = './src/index.js';
 
 const extensions = ['.js', '.jsx'];
 
+const isExternal = id => !id.startsWith('.') && !id.startsWith('/');
```

Comment:
can you please add a comment above this line in the code explaining its purpose? Other than that I think I am good to go!